trimChamber.py

```diff
@@ -57,7 +59,8 @@ def runCommand(cmd):
 
 ohboard = getOHObject(options.slot,options.gtx,options.shelf,options.debug)
 
-dirPath = '%s/%s/trimming/z%f/%s'%(dataPath,chamber_config[options.gtx],ztrim,startTime)
+if options.dirPath == None: dirPath = '%s/%s/trimming/z%f/%s'%(dataPath,chamber_config[options.gtx],ztrim,startTime)
+else: dirPath = options.dirPath
 runCommand(["mkdir","-p",dirPath])
```

Comment:
Let's migrate the ```runCommand``` wrapper out of ```trimChamber``` and use it as an imported function everywhere, i.e., ```run_scans```
I just pushed this to my PR on ```cmsgemos```, can be used a la:
```
from gempython.utils.wrappers import runCommand
...
runCommand(cmd)
```